libraries/AC_PrecLand/AC_PrecLand_Companion.cpp

```diff
@@ -1,6 +1,9 @@
+#include "AC_PrecLand_Companion.h"
+
+#if AP_PREC_LAND_ENABLED
```

Comment:
This is fine, but it would be nice if we could get separate defines in the new `AC_PrecLand_config.h` for each backend.  That file would look something like:

```
#pragma once

#ifndef AC_PRECLAND_ENABLED
#define AC_PRECLAND_ENABLED 1
#endif

#ifndef AC_PRECLAND_BACKEND_DEFAULT_ENABLED
#define AC_PRECLAND_BACKEND_DEFAULT_ENABLED AC_PRECLAND_ENABLED
#endif

#ifndef AC_PRECLAND_COMPANION_ENABLED
#define AC_PRECLAND_COMPANION_ENABLED AC_PRECLAND_BACKEND_DEFAULT_ENABLED
#endif

```
etc etc

Reply:
... to be clear, if this is too much just say so and we can get it done later :-)

Reply:
no problem, now it should be ok, I am checking with `build_all.sh` locally and then I'll push